### ISSUE TITLE
Subtitles make the picture flicker

### ISSUE THREAD

**Issue by Razzeee** [Razzeee/xbmc-test#103]
**Reported by cybearman on 23 Oct 2003 09:59 UTC**

Hi,

I am using xbmc version 21-10-2003 on a v1.0 PAL XBOX 
(16/9 screen)
I have disabled the "autoadjust resolution feature", so 
that each movies plays at the standard PAL resolution.

I get the same problem with every movie which has 
subtitles on more than one line.

Here is the problem:
- In normal zoom mode, I think that the subtitle renderer 
always put the subtitles under the picture, so that. If 
watching for a 2:35 movie, and a one line subtitle, it is 
fine. The problem occurs when a subtitle appears with 
more than one line, I can see the picture jumping a little 
bit up (some pixels probably). Then, the two line subtitle 
disappear. After that, when a one line subtitle appears, 
the picture goes a bit down, taking back its original 
place.
Basically, when you have long and short subtitles, you 
have a jumping pictures, which is kind of annoying.

I guess that the subtitle renderer prefers to make the 
picture jump instead of writing a subtitle on it. 

The problem does not occur neither in stretch mode nor 
in zoom mode.

Thanks for your help,

Patrick

Migrated-From: http://trac.kodi.tv/ticket/103
